samples/unit-tests/indicator-klinger/recalculations/demo.js

```diff
@@ -0,0 +1,306 @@
+            [1553002200000, 126585600],
+            [1553088600000, 124140800],
+            [1553175000000, 204136800],
+            [1553261400000, 169630800],
+            [1553520600000, 175381200],
+            [1553607000000, 199202000],
+            [1553693400000, 119393600],
+            [1553779800000, 83121600],
+            [1553866200000, 94256000],
+            [1554125400000, 111448000],
+            [1554211800000, 91062800],
+            [1554298200000, 93087200],
+            [1554384600000, 76457200],
+            [1554471000000, 74106400],
+            [1554730200000, 103526800],
+            [1554816600000, 143072800],
+            [1554903000000, 86781200],
+            [1554989400000, 83603200],
+            [1555075800000, 111042800],
+            [1555335000000, 70146400],
+            [1555421400000, 102785600],
+            [1555507800000, 115627200],
+            [1555594200000, 96783200],
+            [1555939800000, 77758000],
+            [1556026200000, 93292000],
+            [1556112600000, 70162400],
+            [1556199000000, 74172800],
+            [1556285400000, 74596400],
+            [1556544600000, 88818800],
+            [1556631000000, 186139600],
+            [1556717400000, 259309200],
+            [1556803800000, 127985200],
+            [1556890200000, 83569600],
+            [1557149400000, 129772400],
+            [1557235800000, 155054800],
+            [1557322200000, 105358000],
+            [1557408600000, 139634400],
+            [1557495000000, 164834800],
+            [1557754200000, 229722400],
+            [1557840600000, 146118800],
+            [1557927000000, 106178800],
+            [1558013400000, 132125600],
+            [1558099800000, 131516400],
+            [1558359000000, 154449200],
+            [1558445400000, 113459200],
+            [1558531800000, 118994400],
+            [1558618200000, 146118800],
+            [1558704600000, 94858800],
+            [1559050200000, 111792800],
+            [1559136600000, 113924800],
+            [1559223000000, 84873600],
+            [1559309400000, 108174400],
+            [1559568600000, 161584400],
+            [1559655000000, 123872000],
+            [1559741400000, 119093600],
+            [1559827800000, 90105200],
+            [1559914200000, 122737600],
+            [1560173400000, 104883600],
+            [1560259800000, 107731600],
+            [1560346200000, 73012800],
+            [1560432600000, 86698400],
+            [1560519000000, 75046000],
+            [1560778200000, 58676400],
+            [1560864600000, 106204000],
+            [1560951000000, 84496800],
+            [1561037400000, 86056000],
+            [1561123800000, 191202400],
+            [1561383000000, 72881600],
+            [1561469400000, 84281200],
+            [1561555800000, 104270000],
+            [1561642200000, 83598800],
+            [1561728600000, 124442400],
+            [1561987800000, 109012000]
+        ],
```

Comment:
Can we reduce this? In general unit tests should be as fast/minimal as possible.

Reply:
In the case of that indicator, we have to have about 70 points to calculate each line. I removed some points, but that's all I could do.

Reply:
I see. Thanks!